### PR TITLE
Add @Nickanthonygroen

### PR DESCRIPTION
**Pull request by Nickanthonygroen**

### CODE REVIEW

**Comment by wooorm** [comment]
Cool that you’re using Git! If you do that though, you should link your email to Git and GitHub: https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/

File: site/class-1-play/nickanthonygroen.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 800 400" width="1600" height="800">
+  <!--Update the title with your username-->
+  <title>nickanthonygroen</title>
```
Comment (wooorm) [inline site/class-1-play/nickanthonygroen.svg]: Add `@` before your username.

File: site/class-1-play/nickanthonygroen.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  <!--Add stuff here-->
+  <text x="80" y="800" font-family="Roboto" font-size="30em">
+    ART DECO</text>
+  <circle cx="30" cy="320" r="320" fill="purple" stroke-width="32" stroke="gold" opacity="0.6"/>
+  <g>
```
Comment (wooorm) [inline site/class-1-play/nickanthonygroen.svg]: Your missing `stroke` and `stroke-width` attributes on a `<g>` element

**Comment by wooorm** [comment]
Could you add another commit (e.g., to remove the two comments), when you’ve linked Git and GitHub through your email?

**Comment by Nickanthonygroen** [comment]
@wooorm I've deleted the comments.

**Pull request merged**